app/services/folio_client.rb

```diff
@@ -163,14 +163,30 @@ def as_json
     end
   end
 
+  # TODO: after Poppy launch delete this method
+  def okapi_version
+    response = get('/_/version')
+    check_response(response, title: 'Okapi version', context: {})
+    # response.body is a string
+    response.body
+  end
```

Comment:
```suggestion
  def okapi_version
    @okapi_version ||= begin
      response = get('/_/version')
      check_response(response, title: 'Okapi version', context: {})
      # response.body is a string
      response.body
    end
  end
```

Should we memoize this so we don't keep hitting the version endpoint over and over?